shared/constant/constant.go

```diff
@@ -19,8 +19,13 @@ var PrivilegedPlugins = []string{
 	"plugins/docker",
 	"plugins/gcr",
 	"plugins/ecr",
-	"woodpeckerci/plugin-docker-buildx",
-	"codeberg.org/woodpecker-plugins/docker-buildx",
+	// latest buildx
+	"docker.io/woodpeckerci/plugin-docker-buildx:latest",
+	"codeberg.org/woodpecker-plugins/docker-buildx:latest",
+	// renovate: datasource=docker depName=docker.io/woodpeckerci/plugin-docker-buildx
+	"docker.io/woodpeckerci/plugin-docker-buildx:4.2.0",
```

Comment:
It will be crazy to maintain and you will have to either make this very long or you will be forced to update all pipelines to new versions, or am I missing something? 🤔

Reply:
yes at some point we would force to update the plugin version if a user did pin it to an old one ...
... or he has to explicit allow it